documentation.html

```diff
@@ -703,6 +707,212 @@ <h4>Out of Stock</h4>
           </iframe>
         </section>
         <!--  Cards Section End  -->
+
+        <!--  Images Section Start  -->
+        <section class="component-section" id="images-section">
+          <h4 class="component-heading">Images</h4>
+          <p class="component-sub-heading">
+            Using our library you can easily make your images responsive. Here
+            all the images will be fitted to it's parent width.
+          </p>
+          <p class="component-sub-heading-bold">Example of Responsive Image</p>
+          <p class="component-sub-heading">
+            For using responsive image you can use class
+            <span class="highlight-text">img-responsive</span> this would fit
+            the image to the width of it's parent container.
+          </p>
+
+          <!--  Responsive Image  -->
+          <div
+            class="component-example flex-row flex-row-center-vertical flex-gap-1"
+          >
+            <div style="width: 100%; height: 400px">
+              <img
+                src="https://picsum.photos/1400/400"
```

Comment:
For responsive image I believe you would need to give a src for fallback image and srcset images. So that browers has multiple images and it chooses and shows images according to the dimensions.

Reply:
No here the image would always get fitted to its parent width, and as i have used object-fit property it would lose its aspect ratio.